/controller_node:
  ros__parameters:
    Kp0: 4.1
    Kp1: 4.1
    Kp2: 4.5
    Kv0: 9.0
    Kv1: 9.0
    Kv2: 5.5
    Ki0: 0.0
    Ki1: 0.0
    Ki2: 0.4
    cmd_topic: /fmu/in/vehicle_attitude_setpoint
    ctrl_rate: 50.0
    gra: 9.81
    hov_percent: 0.42
    i_limit_max: 1.5
    i_c: 1.5
    mass: 2.18

